testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -832,6 +832,28 @@ public void testDeleteAllRowsInPartition()
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
+    public void testDeleteAllRowsUnpartitioned()
+    {
+        withTemporaryTable("delete_all_rows", true, false, NONE, tableName -> {
+            onTrino().executeQuery(format("CREATE TABLE %s (id INT) WITH (transactional = true)", tableName));
+
+            log.info("About to insert");
```

Comment:
I agree with @martint on https://github.com/trinodb/trino/pull/9295/files#r714984993

Reply:
All logging removed from this method.